root/static/scripts/release/components/ReleaseRelationshipEditor.js

```diff
@@ -406,14 +407,7 @@ async function submitWorkEdits(
 function* getAllRelationshipEdits(
   state: ReleaseRelationshipEditorStateT,
 ): Generator<
-  {
-    +edits: Generator<
-      [Array<RelationshipStateT>, WsJsEditRelationshipT],
-      void,
-      void,
-    >,
-    +entity: CoreEntityT,
-  },
+  [Array<RelationshipStateT>, WsJsEditRelationshipT],
```

Comment:
We didn't actually use the `entity` property for anything even before this patch, so I simplified the generator type significantly.